func.js

```diff
@@ -18,10 +18,25 @@ async function main() {
     const { downloadDir } = await getLocations();
     
     const versionFile = path.join(downloadDir, 'funcvm-core-tools-version.txt');
+    const localVersionFile = path.join(process.cwd(), '.func-version');
     let version;
 
     if (!!process.env[constants.versionEnvironmentVariableName]) {
         version = process.env[constants.versionEnvironmentVariableName];
+    } else if (fs.existsSync(localVersionFile)) {
+        await new Promise((resolve) => {
+          const re = /(\d+\.\d+\.\d+)( already)? installed./;
+          const p = spawn(`funcvm${process.platform === 'win32' ? '.cmd': ''}`, ['install', fs.readFileSync(localVersionFile, 'utf8').trim()]);
+          p.on('exit', (code)=>{
+            resolve(code);
+          });
+          p.stdout.setEncoding('utf-8');
+          p.stdout.on('data', (data)=>{
+            !/\d+\.\d+\.\d+ already installed./.test(data) && process.stdout.write(data);
+            if (re.test(data)) version = data.match(re)[1];
+          });
+        });
+
```

Comment:
I don't think this is needed. It should just read the file and set `version`. The error message below will tell the customer how to install it if it's missing.

Reply:
I will make these codes more simply based on your comment.